### PR TITLE
vote: CabinfeverB as pd comitter

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### CODE REVIEW

**Review by bufferflies — APPROVED** [review]
LGTM

File: votes/0186-CabinfeverB-as-pd-committer.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+See more:
+* [Authored pull requests](https://github.com/tikv/pd/pulls?q=is%3Apr+author%3ACabinfeverB+is)
+
+I ([@lhy1024](https://github.com/lhy1024)) hereby nominate [@CabinfeverB](https://github.com/CabinfeverB) as PD Committer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 72 hours unless there is an objection or not enough votes.
+
+## Result
```
Comment (rleungx) [inline votes/0186-CabinfeverB-as-pd-committer.md:20]: The result should be updated after the voting is finished.

**Review by rleungx — APPROVED** [review]

**Review by nolouch — COMMENTED** [review]
+1

File: votes/0186-CabinfeverB-as-pd-committer.md (hunk 1 above)
Comment (lhy1024) [reply votes/0186-CabinfeverB-as-pd-committer.md:20]: done

**Review by nolouch — APPROVED** [review]

**Pull request merged**